### ISSUE TITLE
Solus 4.5 Failure to restore session after Hibernate

### ISSUE THREAD

**Issue by WetGeek** [getsolus/packages#1014]
### Summary

I've created a VM using the 4.5 Plasma .iso and configured it to represent my daily driver laptop.  That includes a top bar and a latte dock.  There are 9 virtual desktops.  With an application in each desktop, I've tried many times, using several methods to hibernate that VM, but it always comes back with all 9 desktops empty.

I've tried creating a custom command based on CTRL+H (systemctl hibernate), as I use on my laptop and all my other VMs, but executing that command simply logs out instead of hibernating.  I've tried issuing that command in a terminal, and I've tried selecting from the main menu.  In every case, the result is a reboot, not a hibernate.

### Steps to reproduce

1.  Create a VM using VBox, and based on the 4.5 Plasma .iso.
2. Configure it to include multiple virtual desktops.
3. Start a different application in each virtual desktop.
4. Attempt to hibernate the VM using any method mentioned in the Summary, above.
5. Restart the VM.

### Expected result

I expected the VM to be restored, with the virtual desktops still containing their applications.

### Actual result

The virtual desktops are all empty, indicating that a reboot was performed, not a hibernate.

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

Plasma

### System details

![image](https://github.com/getsolus/packages/assets/6805994/213cbea7-1c7c-4b10-9872-df66b18a34e0)

**Comment by WetGeek** [comment]
I forgot to mention that the VM was created with 4 GB of RAM and 6 GiB of swap partition.  That's the standard I've used to create 6 other current VMs that all hiberate correctly.  I created the partitions with KDE Partition Manager before installing the OS, becuase my attempt to do that during the installation was very confusing and unsuccessful.  Probably because of errors on my behalf, but I wanted to get the VM built, so I did it my way.

I'll try preparing the partitions using the installer again when I create a VM from the next RC.

**Comment by ReillyBrogan** [comment]
Are you sure that this actually works in VMs that are installed from 4.4 and fully updated? I actually can't get it to work in either case

**Comment by WetGeek** [comment]
The Xfce 4.5 VM that I just created has a hibernation problem as well (see my forum post).  But that VM was just created this morning, and I haven't worked with it much, or even rebooted it.  If I find out that it's a persistent issue, I'll create a bug report.

**Comment by WetGeek** [comment]
After further checking, it turns out that ALL four of the VMs that I created from RC1 .ISO files (and fully updated with each sync) exhibit the same error, not just the Plasma DE.  Furthermore, the Latitude laptop that I'd ordered for testing arrived at a later date, and I installed RC 3 and later RC 4 on that machine.  It also exhibits the same behavior.  That is, it also appears to hibernate correctly, but upon restarting none of the previous session is restored.  In effect, for all these 4.5 installations, the behavior of "hibernate" is the same as the behavior of "reboot."

**Comment by WetGeek** [comment]
Somewhere between RC1 and RC4, this issue was resolved.  I'm not sure what actually fixed it, just that it IS fixed.  I've now installed Xfce RC4 on my Testing laptop (using Calamares to do the partitioning), and configured to to be as nearly a match for my 4.4 Plasma daily-driver laptop as possible (fiven that they're two different DEs), and I've been able to hibernate it without a problem and it recovers the existing session perfectly upon a restart.  I've also built a new 4.5 Plasma VM, again, using Calamares to do the partitioning, and that also performs perfectly.

So I'm now convinced that the issues mentioned in this bug report have been resolved, and I'll now close this as completed.

**Comment by WetGeek** [comment]
Somewhere between RC1 and RC4, this issue was resolved. I'm not sure what actually fixed it, just that it IS fixed. I've now installed Xfce RC4 on my Testing laptop (using Calamares to do the partitioning), and configured to to be as nearly a match for my 4.4 Plasma daily-driver laptop as possible (fiven that they're two different DEs), and I've been able to hibernate it without a problem and it recovers the existing session perfectly upon a restart. I've also built a new 4.5 Plasma VM, again, using Calamares to do the partitioning, and that also performs perfectly.

So I'm now convinced that the issues mentioned in this bug report have been resolved, and I'll now close this as completed.

**Comment by WetGeek** [comment]
Somewhere between RC1 and RC4, this issue was resolved. I'm not sure what actually fixed it, just that it IS fixed. I've now installed Xfce RC4 on my Testing laptop (using Calamares to do the partitioning), and configured to to be as nearly a match for my 4.4 Plasma daily-driver laptop as possible (fiven that they're two different DEs), and I've been able to hibernate it without a problem and it recovers the existing session perfectly upon a restart. I've also built a new 4.5 Plasma VM, again, using Calamares to do the partitioning, and that also performs perfectly.

So I'm now convinced that the issues mentioned in this bug report have been resolved, and I'll now close this as completed.
